### PR TITLE
TrackerCommonData: Categorization script update

### PR DESCRIPTION
**Pull request by vargasa**
#### PR description:

`TrackerCommonData/data/Materials` package provides a script for building the categorization maps required for running `Validation/Geometry` module. This script was *a little bit* outdated (FORTRAN77) and here it is being rewritten to python3 with the same functionality (except the creation of `pdf` files, information which is now being printed directly on the console).

#### PR validation:

This update was proposed in a [Tracker DPG Meeting](https://indico.cern.ch/event/787654/contributions/3298998/attachments/1786295/2908498/Material_Budget_Categorization.pdf) and results in [the Material budget](https://indico.cern.ch/event/787656/contributions/3331625/attachments/1801554/2938673/FortranToPython_MaterialBudget.pdf) meeting. Validation was performed by comparing the output of pdf files and the new code. We'll keep the FORTRAN file until a full update of the documentation has been done.

### CODE REVIEW

**Comment by cvuosalo** [comment]
@cmsbuild please test

**Comment by cvuosalo** [comment]
@cmsbuild please test

**Comment by vargasa** [comment]
@cvuosalo Any comments on this? Pls let me know

**Comment by cvuosalo** [comment]
+1

File: Geometry/TrackerCommonData/data/Materials/mixture.py [hunk 1]
```diff
@@ -0,0 +1,385 @@
+#!/bin/python3
```
Comment (fabiocos) [inline Geometry/TrackerCommonData/data/Materials/mixture.py]: @vargasa why not the usual #!/usr/bin/env python3 , provided we want to force python3?
@davidlange6 FYI, as we are in the middle of the migration

**Comment by fabiocos** [comment]
@vargasa why is this script under data ? I do not want to play the purist, but it does not seem to me the best location. I see you have there a few standalone utilities, may be a bin subdirectory would be more appropriate?

**Comment by cvuosalo** [comment]
Why is this Python script not in the `python` directory?

**Comment by fabiocos** [comment]
@vciulli as it is not a configuration fragment but a standalone utility I would suggest to put it into a bin subdirectory

**Comment by vargasa** [comment]
> @vargasa <https://github.com/vargasa> why is this script under data ? 

I'd say historical reasons. The Fortran script has been here since 
untraceable times. The idea is to replace the fortan script that as you
said is an standalone utility.

> do not want to play the purist, but it does not seem to me the best 
> location. I see you have there a few standalone utilities, may be a bin 
> subdirectory would be more appropriate?

I agree with you. However, it will require updating twiki documentation, 
that I may not be aware of. Will you agree if we let it go through and 
afterwards think about a better directory structure for the module?

**Comment by cvuosalo** [comment]
+1

**Comment by kpedro88** [comment]
please test

**Comment by kpedro88** [comment]
+1

**Comment by vargasa** [comment]
Regarding moving utilities to a new folder. Scripts will be moved from: 

```
Geometry/TrackerCommonData/data/Materials/
|-- Makefile
|-- insertMaterial.py
|-- mixture.f
|-- mixture.py
|-- mixture_weightfractions.f
|-- tex2xml.pl
|-- twikiConfig.ini
`-- twikiExport.py
```
to
```
Geometry/TrackerCommonData/bin/
```
As suggested by @fabiocos. All the other files `*.in` and `*.input` contain actual `data/Material` info. @fabiocos, @cvuosalo, @davidlange6 Any other suggestions/comments regarding this?

**Comment by cvuosalo** [comment]
@vargasa: Yes, `bin` seems to be the best location for them.

**Comment by fabiocos** [comment]
@vargasa thanks, I think it is a more clear structure

**Pull request merged**